diff -urN libretro-20170303/libretro-vba_next/src/port.h libretro-20170303-patched/libretro-vba_next/src/port.h
--- libretro-20170303/libretro-vba_next/src/port.h	2017-03-03 21:15:23.000000000 +1000
+++ libretro-20170303-patched/libretro-vba_next/src/port.h	2017-03-04 00:07:42.315844212 +1000
@@ -88,16 +88,6 @@
 #define WRITE32LE(x,v) *((u32 *)(x)) = (v)
 #endif
 
-#ifdef INLINE
- #if defined(_MSC_VER)
-  #define FORCE_INLINE __forceinline
- #elif defined(__GNUC__)
-  #define FORCE_INLINE inline __attribute__((always_inline))
- #else
-  #define FORCE_INLINE INLINE
- #endif
-#else
- #define FORCE_INLINE
-#endif
+#define FORCE_INLINE
 
 #endif // PORT_H
